docs/users.md

```diff
@@ -13,6 +15,8 @@ This is a collection of user methods.
 ## `sdk.users.me()`
 This method is used to get details about the current logged in user.
 
+**[Documentation](user-get-me)**
```

Comment:
I want something to clarify that it's main API Documentation, but that'll be confused with `API` right below.

What if we did something like:

```
Read [`/users/me` documentation](user-get-me) for more details.
```

?

Reply:
Yes that sounds good.